vuepress/docs/next/tutorials/getting-started/arm-install.md

```diff
@@ -0,0 +1,265 @@
+pn-3c07adf0-fac54a9f-entando-app-builder-menu-bff-deploymep6j87                 0/1     Pending             0          5s
+pn-3c07adf0-fac54a9f-entando-app-builder-menu-bff-deploymep6j87                 0/1     ContainerCreating   0          5s
+pn-3c07adf0-fac54a9f-entando-app-builder-menu-bff-deploymep6j87                 0/1     Running             0          40s
+pn-3c07adf0-fac54a9f-entando-app-builder-menu-bff-deploymep6j87                 0/1     Running             0          53s
+pn-3c07adf0-fac54a9f-entando-app-builder-menu-bff-deploymep6j87                 1/1     Running             0          54s
+pn-3c07adf0-fac54a9f-entando-app-builder-menu-bff-deployer-6999                 0/1     Completed           0          69s
+quickstart-pn-3c07adf0-fac54a9f-entando-app-builder-menu-b-link-deployer-2017   0/1     Completed           0          70s
+pn-3c07adf0-fac54a9f-entando-app-builder-menu-bff-deployer-6999                 0/1     Completed           0          71s
+pn-3c07adf0-fac54a9f-entando-app-builder-menu-bff-deployer-6999                 0/1     Terminating         0          71s
+pn-3c07adf0-fac54a9f-entando-app-builder-menu-bff-deployer-6999                 0/1     Terminating         0          71s
+quickstart-pn-3c07adf0-fac54a9f-entando-app-builder-menu-b-link-deployer-2017   0/1     Completed           0          72s
+quickstart-pn-3c07adf0-fac54a9f-entando-app-builder-menu-b-link-deployer-2017   0/1     Terminating         0          72s
+quickstart-pn-3c07adf0-fac54a9f-entando-app-builder-menu-b-link-deployer-2017   0/1     Terminating         0          72s
+
+```
+
+</details>
+
+---
+
+<details><summary>What Pods come out of the box?</summary>
+
+``` bash
+kubectl get pods -n YOUR-NAMESPACE
+```
+
+``` shell-session
+NAME                                                  READY   STATUS    RESTARTS   AGE
+entando-operator-cf499c46c-r2bhn                                  1/1     Running   0          13m
+entando-k8s-service-6c94495c87-j9p7b                              1/1     Running   0          13m
+default-postgresql-dbms-in-namespace-deployment-ddf96f9c6-k46w4   1/1     Running   0          13m
+default-sso-in-namespace-deployment-59d6cd795b-pgvth              1/1     Running   0          13m
+quickstart-ab-deployment-7cd5c9987b-v96sv                         1/1     Running   0          13m
+quickstart-cm-deployment-5896d4f8c5-sl8pg                         1/1     Running   0          13m
+quickstart-deployment-57864cdfdf-n699w                            1/1     Running   0          13m
+pn-3c07adf0-fac54a9f-entando-app-builder-menu-bff-deploymep6j87   1/1     Running   0          13m
+```
+
+</details>
+
+---
+
+## Step 4. Log in to Entando
+
+Now that you've installed Entando, log in to the Entando App Builder.
+
+The URL of your running Entando instance is http://ingressHostName/app-builder/.
+
+
+- Example URL
+
+``` bash
+quickstart.192.168.64.33.nip.io/app-builder/
```

Comment:
@pietrangelo, is this a good example for the App Builder URL. I tried to test this tutorial but could not install the entando resources (thinking it's my profile) and had to move on to a marketing document with a deadline.

Reply:
@jyunmitch yes it is.